yb-voyager/src/srcdb/postgres.go

```diff
@@ -272,3 +272,17 @@ func (pg *PostgreSQL) FilterEmptyTables(tableList []*sqlname.SourceName) ([]*sql
 	}
 	return nonEmptyTableList, emptyTableList
 }
+
+func (pg *PostgreSQL) IsTablePartition(table *sqlname.SourceName) bool {
+	var parentTable *sqlname.SourceName
+	query := fmt.Sprintf(`SELECT inhparent::pg_catalog.regclass
+	FROM pg_catalog.pg_class c JOIN pg_catalog.pg_inherits ON c.oid = inhrelid
+	WHERE c.oid = '%s'::regclass::oid`, table.Qualified.Unquoted)
+
+	err := pg.db.QueryRow(context.Background(), query).Scan(&parentTable)
```

Comment:
I don't think you will be able to extract the parent table name in a variable of type `*sqlname.SourceName`. You will need a plain string.